nlp-engine/auto_trial.py

```diff
@@ -0,0 +1,140 @@
+import nltk
+import string
+from nltk.corpus import brown
+from nltk.corpus import wordnet
+from nltk.util import ngrams
+from fuzzywuzzy import fuzz
+import requests
+import json
+
+#stopwords=open('stopwords.txt').readlines()
+r_stop=requests.get('https://raw.githubusercontent.com/egovernments/egov-mdms-data/DEV/data/pb/Chatbot/stopwords.txt')
+stopwords=r_stop.text.splitlines()
+
+#for i in range(0,len(stopwords)):
+#    stopwords[i]= stopwords[i][0:len(stopwords[i])-1].lower()
+
+rem=stopwords
+#rem= rem+['show','see']
+keep=['are','to','be','not']
+from autocorrect import Speller
+spell=Speller(lang='en')
+
+r= requests.get('https://raw.githubusercontent.com/egovernments/egov-mdms-data/DEV/data/pb/Chatbot/KeyWords.json')

```

Comment:
Don't hardcode, the link is only for dev env

Reply:
This file is renamed as Preprocessing.py in src. Please check that.